### DISCUSSION TITLE
Is there a way to disable the debugger/log window on a program compiled as a debug build, without access to source code?

### DISCUSSION THREAD

**Discussion by ImAburger** [HaxeFlixel/flixel#2895] [Q&A]
There's a Friday Night Funkin' mod that I want to edit, and plan to release online once I'm finished.. The only issue is that the build of the mod I have is a debug build, and I'd like to have the debugging tools not be visible ingame, since I intend for other people play my version of it. I don't have access to the source code of the mod to compile my own release build, and there isn't anybody I can ask who does. I'm not sure if what I'm asking is even possible, which is why I came here to ask all of you. Thank you so much in advance!

**Comment by moxie-coder** [comment]
you can do `FlxG.debugger.drawDebug = false;` or `FLX_NO_DEBUG` in Project.xml

**Reply by Snirozu** [reply]
> without access to source code

**Reply by moxie-coder** [reply]
then you can't unless you can run `FlxG.debugger.drawDebug = false;` in console, otherwise you can't

**Comment by ImAburger** [comment]
> you can do `FlxG.debugger.drawDebug = false;` or `FLX_NO_DEBUG` in Project.xml

I don't have access to the source code, just the exported debug build, so I don't think I can do that.

**Reply by moxie-coder** [reply]
well then you can't

**Reply by moxie-coder** [reply]
I mean, you could use the console to run `FlxG.debugger.drawDebug = false;` but otherwise you can't

**Comment by Geokureli** [comment]
maybe contact the people who made it and ask for one? otherwise no

**Comment by ImAburger** [comment]
> I mean, you could use the console to run `FlxG.debugger.drawDebug = false;` but otherwise you can't

I tried running that code in the console, but it just returns "> false" and does nothing else. Am I doing something wrong?

**Reply by Geokureli** [reply]
reply to whatever post you're referring to

**Reply by moxie-coder** [reply]
he was referring to me

**Reply by Geokureli** [reply]
I know, I'm saying they should reply to your answer rather than creating a new answer

**Reply by moxie-coder** [reply]
alright

**Reply by ImAburger** [reply]
sorry, this is my first time posting on this site, so i'm not very familiar with this ui 😭
